arangod/Cluster/ClusterInfo.cpp

```diff
@@ -2441,15 +2469,13 @@ int ClusterInfo::ensureIndexCoordinator(
     do {
       resultBuilder.clear();
       errorCode = ensureIndexCoordinatorWithoutRollback(
-        databaseName, collectionID, idString, slice, create,
-        resultBuilder, errorMsg, timeout);
+        databaseName, collectionID, idString, slice, create, resultBuilder,
+        errorMsg, timeout);
       
-      if (errorCode == TRI_ERROR_NO_ERROR) {
-        return errorCode; // done
-      } else if (errorCode == TRI_ERROR_HTTP_PRECONDITION_FAILED) {
+      if (errorCode == TRI_ERROR_HTTP_PRECONDITION_FAILED) {
```

Comment:
This looks better as it is right now in devel. Probably it was changed there since you last merged?

Reply:
is correct behaviour exits after 120s trying to put the index in plan.